legacyCore/src/main/java/com/hamdan/forzenbook/legacy/core/view/NavigatorImpl.kt

```diff
@@ -0,0 +1,23 @@
+package com.hamdan.forzenbook.legacy.core.view
+
+import android.content.Context
+
+class NavigatorImpl(
+    val legacyLoginContext: Context,
+    val legacyCreateAccountContext: Context,
+) : Navigator {
+    // Todo implement both when FA-100 and 102 are implemented
+    override fun navigateToLogin() {
+// may want to change this from implicit intent incase we want to fill in email other options are possible too
```

Comment:
Any sort of implicit sharing would definitely be new and different functionality

Reply:
haha, mixed up explicit and implicit